Cargo.toml

```diff
@@ -51,5 +51,5 @@ itertools = "0.10"
 colored = "2.1.0"
 dprint-plugin-typescript = "0.91.1"
 dprint-vue-plugin = "0.1.0"
-git2 = "0.19.0"
+git2 = { version = "0.19.0", default-features = false, features = ["https", "ssh_key_from_memory", "vendored-libgit2", "vendored-openssl"] }
```

Comment:
I've enabled vendoring where possible on Holochain crates because not depending on the library that happens to have been provided is more likely to build right. Like when somebody does a `cargo install` later and links against some openssl that has been built with different flags than these libraries expect - can cause some really tricky to understand build problems. 

If we don't need this change I'm okay with reverting it, but even if we don't need it, it's still a preference to build everything from source at versions we know have been tested together.

Reply:
I see, this makes sense.